avaframe/com1DFA/com1DFA.py

```diff
@@ -89,13 +91,15 @@ def com1DFAMain(avalancheDir, cfgMain, cfgFile='', relThField='', variationDict=
 
     # create a list of simulations and generate an individual configuration object for each simulation
     # if need to reproduce exactly the hash - need to be strings with exactely the same number of digits!!
-    simDict = com1DFA.prepareVarSimDict(modCfg, inputSimFiles, variationDict)
+    simDFOld, simNameOld = cfgUtils.readAllConfigurationInfo(avalancheDir, specDir='')
```

Comment:
add a comment that first info about the configuration of all existing sims is fetched and a DF is created